package.json

```diff
@@ -4,6 +4,7 @@
     "packages/@okta/*"
   ],
   "scripts": {
+    "preinstall": "npm config delete cafile",
```

Comment:
What impact does this have to someone that has an existing .npmrc config outside of this module?   (does it change that .npmrc file?)

Reply:
It does not change any files. The setting seems to remain in effect (globally) possibly until I restart my machine. `npm config get cafile`

Reply:
Wrong about that. It definitely removes it from the file.

Reply:
In that case, I'm definitely not in favor of this change in a repo that non-okta users can clone.  Thanks for checking!